src/test/java/io/kestra/plugin/aws/glue/StopJobRunIntegrationTest.java

```diff
@@ -0,0 +1,75 @@
+package io.kestra.plugin.aws.glue;
+
+import io.kestra.core.junit.annotations.KestraTest;
+import io.kestra.core.models.property.Property;
+import io.kestra.core.runners.RunContextFactory;
+import io.kestra.plugin.aws.glue.model.Output;
+import jakarta.inject.Inject;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@KestraTest
+@Disabled("Provide AWS credentials, an existing Glue job name, and a job run ID to stop to run this test")
+class StopJobRunIntegrationTest {
```

Comment:
It's IT by convention:

```suggestion
class StopJobRunIT {
```